Source/WTF/wtf/SuspendableWorkQueue.h

```diff
@@ -40,14 +40,22 @@ class SuspendableWorkQueue final : public WorkQueue {
     WTF_EXPORT_PRIVATE static Ref<SuspendableWorkQueue> create(const char* name, QOS = QOS::Default, ShouldLog = ShouldLog::No);
     WTF_EXPORT_PRIVATE void suspend(Function<void()>&& suspendFunction, CompletionHandler<void()>&& suspensionCompletionHandler);
     WTF_EXPORT_PRIVATE void resume();
-    WTF_EXPORT_PRIVATE void dispatch(Function<void()>&&) final;
+    void dispatch(Function<void()>&& function) final
```

Comment:
I'm not sure these virtual functions should be defined inline.
The issue is that libWTF.a has to have an definition, but then the use-sites in WebCore.framework might also generate a definition. Seems a bit too risky for what it presumably tries to achieve?

Reply:
I made a note on why it's done that way in the commit log, and discussion is on slack. Issue is with a JavaScriptCore utilities reading symbols, it gives error when that particular method isn't inline. no idea why.

Reply:
FWIW, it seems to be a bug in the `tapi` utility